src/transformStateWithClones.js

```diff
@@ -7,7 +7,35 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const stateAfterActions = [];
```

Comment:
It is not a bad one, but a bit overcomplicated. We can make it simpler and clearer:
```suggestion
  const states = [];
```
or even better:
```suggestion
  const stateVersions = [];
```